README.md

```diff
@@ -4,13 +4,15 @@ Returns active streamers on Anrop.
 
 ## Requirements
 
-Code is written in [Go](https://golang.org/) and uses [gb](https://getgb.io/) to compile.
+Code is written in [Go](https://golang.org/) and uses
+[Go modules](https://github.com/golang/go/wiki/Modules) for dependency
+management.
 
 ## How To Use
 
-Compile the sources with `gb build`.
+Download all dependencies and compile the sources with `go build`.
```

Comment:
This makes it seem like you have to download the dependencies separately.
```suggestion
Use `go build` to download all dependencies and compile the sources.
```
Or something like that?